src/services/bulkstatus.service.ts

```diff
@@ -57,227 +82,123 @@ async function checkBulkStatus(req, res) {
     res.set('X-Progress', `${(percentComplete * 100).toFixed(2)}% Done`);
     res.set('Retry-After', 120);
   } else if (bulkStatus.status === 'Failed') {
+    // Not sure what the response should be - adding an OperationOutcome to an errors.ndjson
+    // file for now
     logger.debug(`bulkStatus entry is failed`);
     res.status(200);
 
-    // Create FHIR Bundle response
-    response = {
-      resourceType: 'Bundle',
-      type: 'batch-response',
-      entry: [
-        {
-          response: {
-            status: '400',
-            outcome: {
-              resourceType: 'OperationOutcome',
-              issue: [
-                {
-                  severity: 'fatal',
-                  code: 'transient',
-                  details: {
-                    text: bulkStatus.error.message ?? "Internal System Error: '$import' request not processed."
-                  }
-                }
-              ]
+    const operationOutcomeOutput: fhir4.OperationOutcome[] = [
+      {
+        resourceType: 'OperationOutcome',
+        issue: [
+          {
+            severity: 'fatal',
+            code: 'transient',
+            details: {
+              text: bulkStatus.error.message ?? "Internal System Error: '$import' request not processed."
             }
           }
-        }
-      ]
-    };
+        ]
+      }
+    ];
+
+    writeToFile(operationOutcomeOutput, clientId, false);
 
-    return response;
+    exportManifest.error.push({
+      extension: {
+        manifestUrl: bulkStatus.manifestUrl
+      },
+      url: `http://${process.env.SERVER_HOST}:${process.env.SERVER_PORT}/${req.params.base_version}/file/${clientId}/errors.ndjson`
```

Comment:
We construct the `http://${process.env.SERVER_HOST}:${process.env.SERVER_PORT}/${req.params.base_version}/file/${clientId}` part of the url enough times that maybe it's worth creating a shorthand for it at the top of the function?